docs/samples/git-setup/clustertemplates/version_4.Y/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v1
```

Comment:
why not include z number as well? with y only, it could still be a little confusing to remember v4-16-1/2/3 is ocp 4.16.5 and 4 is 4.16.10 etc. Plus it makes it obvious when you update the ref in provisioning request, whether it's gonna a release upgrade.

Reply:
Thanks, @bartwensley and @jc-rh, I'll go with `sno-ran-du.v4-Y-Z-1` for now then as an example and we can update it later if needed as the upgrades work progresses.